packages/eventual-send/src/postponed.js

```diff
@@ -0,0 +1,48 @@
+// @ts-check
+/// <reference types="ses" />
+
+/**
+ * @template T
+ * @typedef {import('.').EHandler<T>} EHandler
+ */
+
+/**
+ * @param {import('.').HandledPromiseConstructor} HandledPromise
+ * @returns {[Required<EHandler<any>>, () => void]}
+ */
+export const makePostponedHandler = HandledPromise => {
+  // Create a simple postponedHandler that just postpones until the
+  // fulfilledHandler is set.
+  let donePostponing;
+  const interlockP = new Promise(resolve => {
+    donePostponing = () => resolve(undefined);
+  });
+
+  const makePostponedOperation = postponedOperation => {
+    // Just wait until the handler is resolved/rejected.
+    return function postpone(x, ...args) {
+      // console.log(`forwarding ${postponedOperation} ${args[0]}`);
+      return new HandledPromise((resolve, reject) => {
+        interlockP
+          .then(_ => {
+            // If targetP is a handled promise, use it, otherwise x.
```

Comment:
With the code moved here, this is the only mention of `targetP` in this file, which is confusing. Same with the `fulfilledHandler` above, but somehow I was less confused by that one.